### PR TITLE
Add an optional repeat parameter to udpateJob()

### PR DESCRIPTION
**Pull request by tgolen**
I'm not sure what the procedure is here, sooooooooooo

### CODE REVIEW

**Comment by tgolen** [comment]
Also, the matching bedrock PR has already been merged 

https://github.com/Expensify/Bedrock/pull/95

File: src/Jobs.php [hunk 1]
```diff
@@ -142,16 +142,18 @@ public function getJob($name, $timeout = 0)
...
      */
-    public function updateJob($jobID, $data)
+    public function updateJob($jobID, $data, $repeat = null)
     {
         return $this->call(
             "UpdateJob",
             [
                 "jobID"    => $jobID,
                 "data"     => $data,
+                "repeat"   => $repeat,
```
Comment (quinthar) [inline src/Jobs.php]: Does this work if `$repeat` is null?  Wouldn't it send a blank repeat and trigger a `402 Malformed repeat`?

Comment (tgolen) [inline src/Jobs.php]: I think null is OK. It's used for all the optional parameters for all the other methods.

Comment (tgolen) [inline src/Jobs.php]: Especially `createJob()` which has the same bedrock validation for the `repeat` param.

Comment (quinthar) [inline src/Jobs.php]: Oh, cool.

**Comment by quinthar** [comment]
Ok I've merged it, but I'm not sure how we're deploying this.  I think there is a composer package somewhere, but I don't know what the next step is.  @righdforsa What's next here?

**Comment by tgolen** [comment]
I think we need to create a tag, and then update `composer.json` in the web repo to point to the new tag.

**Comment by iwiznia** [comment]
Oh, shoot... Yeah, you needed to run the `package.sh` script of this repo, which would change the version number and create a tag. Once you do that, you need to change the version number in composer.json of web-e and run `../script/composer.sh update expensify/php-libs`

**Pull request merged**